backend/src/csv_handler/auth/verifier.py

```diff
@@ -11,7 +11,7 @@
 from fastapi.security import HTTPBearer
 from typing import Tuple
 from utils.exceptions import RegionIsNotSet, UserPoolIdIsNotSet, AppClientIdIsNotSet
-from utils.logger import log
+from utils.logger import log, console
```

Comment:
Still importing console
```suggestion
from utils.logger import log
```